subjects/java/checkpoints/config-protector/ExerciseRunner.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Arrays;
+
```

Comment:
It's missing this import:
`import java.util.List;`

Reply:
fixed